evm/evm-typegen/src/pretty-out-dir.ts

```diff
@@ -0,0 +1,24 @@
+import { FileOutput, OutDir } from "@subsquid/util-internal-code-printer";
+import * as prettier from "prettier";
```

Comment:
Let's prettify files by hands. I don't think it's that important to use prettier for that